Doc/library/datetime.rst

```diff
@@ -14,40 +14,59 @@
 
 .. XXX what order should the types be discussed in?
 
-The :mod:`datetime` module supplies classes for manipulating dates and times in
-both simple and complex ways.  While date and time arithmetic is supported, the
-focus of the implementation is on efficient attribute extraction for output
-formatting and manipulation. For related functionality, see also the
-:mod:`time` and :mod:`calendar` modules.
+The :mod:`datetime` module supplies classes for manipulating dates and times.
 
-There are two kinds of date and time objects: "naive" and "aware".
+While date and time arithmetic is supported, the focus of the implementation is
+on efficient attribute extraction for output formatting and manipulation.
 
-An aware object has sufficient knowledge of applicable algorithmic and
+.. seealso::
+
+   Module :mod:`calendar`
+      General calendar related functions.
+
+   Module :mod:`time`
+      Time access and conversions.
+
+   Package `dateutil <https://dateutil.readthedocs.io/en/stable/>`_
+      Third-party library with expanded time zone and parsing support.
+
+.. _datetime-naive-aware:
+
+Naive vs Aware Objects
+----------------------
+
+There are two kinds of date and time objects: "naive" and "aware."
+
+An **aware** object has sufficient knowledge of applicable algorithmic and
 political time adjustments, such as time zone and daylight saving time
-information, to locate itself relative to other aware objects.  An aware object
+information, to locate itself relative to other aware objects. An aware object
 is used to represent a specific moment in time that is not open to
 interpretation [#]_.
 
-A naive object does not contain enough information to unambiguously locate
-itself relative to other date/time objects.  Whether a naive object represents
+A **naive** object does not contain enough information to unambiguously locate
+itself relative to other date/time objects. Whether a naive object represents
 Coordinated Universal Time (UTC), local time, or time in some other timezone is
 purely up to the program, just like it is up to the program whether a
-particular number represents metres, miles, or mass.  Naive objects are easy to
+particular number represents metres, miles, or mass. Naive objects are easy to
```

Comment:
```suggestion
particular number represents meters, miles, or mass. Naive objects are easy to
```

(I believe Python prefers US spelling...)

Reply:
Nitpicking – and I know here you only reformatted, but `meters`, `miles` are units, `mass` is a dimension...should maybe be k.g for consistency ??

Reply:
Yes, I had noticed the non-US units & spelling but was unsure of the preferred convention so left as-is.

Reply:
Regarding this:

1. I am indifferent.
2. I don't see anything siding in either direction in https://devguide.python.org/documenting/#style-guide
3. A quick `egrep -nr 'meter' Doc | grep -vi parameter` versus `egrep -nr 'metre' Doc` shows uses of "meters" in enum.rst and uses of "metres" in tkinter.rst.  It seems the court is still out.